### DISCUSSION TITLE
How to migrate MonitorControl settings to a new machine

### DISCUSSION THREAD

**Discussion by usizu** [MonitorControl/MonitorControl#1203] [Q&A] [answered]
Hi,

What is the best way to migrate MonitorControl settings to a new Mac? Is there a single preferences/config file that is kept somewhere that can be easily copied?

Thanks and apologies if this is already answered somewhere but I couldn't find it

**Comment by waydabber** [comment, marked as answer]
Yes, the config file is under ~/Library/Preferences/me.guillaumeb.MonitorControl.plist. You can transplant this to your new Mac.

**Reply by usizu** [reply]
Great, thanks!
